### PR TITLE
HAWKULAR-1326 Create prometheus config folder before cache init

### PR DESCRIPTION
**Pull request by lucasponce**
- Change target folder

### CODE REVIEW

**Comment by lucasponce** [comment]
Please @jshaughn, before to merge, let me know if it you can test it and check your scenario.
Thx

File: hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryConfig.java [hunk 1]
```diff
@@ -75,6 +76,9 @@
     public InventoryConfig() {
         configPath = Paths.get(System.getProperty("jboss.server.config.dir"), "hawkular");
```
Comment (jmazzitelli) [inline hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryConfig.java]: while we are doing mkdirs in this method, perhaps we should also mkdirs() this hawkular directory just to be safe?

Comment (lucasponce) [reply hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryConfig.java]: Yes, we can protect here and perform a mkdirs() too.

**Comment by lucasponce** [comment]
@jmazzitelli I addressed your comments, I will merge it and bump services with the SCRDEPS and the change on run-prometheus.sh script.

**Pull request merged**